src/client.ts

```diff
@@ -241,16 +241,11 @@ export class BalenaCertificateClient {
 
 		return greenlockInst
 			.check({ domains: requestDomains })
-			.then(function(results) {
+			.then(function(certsExist) {
```

Comment:
Wht not arrow function?

Reply:
As with Akis' comments, my brain must have hotswitched. It should indeed be an arrow function!